docs/release-notes/8.0/8.0.35-31.0.md

```diff
@@ -0,0 +1,33 @@
+# Percona XtraBackup 8.0.35-31 (2024-06-26)
+
+Get started with [Quickstart Guide for Percona XtraBackup](..//..//quickstart-overview.md).
+
+## Release highlights
+
+This release adds Percona XtraBackup ARM64 packages and provides bug fixes.
+
+## Bug fix
+
+* [PXB-3283]: When XtraBackup performs a backup and exports a tablespace for tables that have changed their charset-collation before the backup, it creates an incorrect table definition. This incorrect table definition means the table's structure information may not match the data.
+
+* [PXB-2797]:  When you import a table with a full-text index from an XtraBackup backup, you might encounter a schema mismatch. This happens when the table's structure in the backup does not match the structure expected by the database.
+
+* [PXB-3210]: Percona XtraBackup version 8.0.33-28 and later fail to build on MacOS because of the FIND_PROCPS() function. This function causes issues during the build process, preventing the software from compiling successfully.
+
+* [PXB-3130]: During an upgrade, if a type is out of bounds, the Prepare step fails. This means that a value exceeds the expected range, causing the upgrade process to stop and not complete.
```

Comment:
when preparing 8.0.30 backups with newer xtrabackup versions, prepare step failed.

Reply:
And please add thank you note to the contributor LiNing and githubid: @LI8023NING